src/PositionManager.sol

```diff
@@ -432,8 +430,7 @@ contract PositionManager is
         liquidity = poolManager.getPositionLiquidity(config.poolKey.toId(), positionId);
     }
 
-    /// @inheritdoc IPositionManager
-    function getPositionConfigId(uint256 tokenId) external view returns (bytes32) {
-        return positionConfigs[tokenId].getConfigId();
+    function _toConfig(PositionInfo memory info) internal view returns (PositionConfig memory config) {
```

Comment:
should PositionInfo by a library with this function in it?
im open

Reply:
yeah was gonna move some things around just wanted to check people liked the basic idea